framework/include/variables/MooseVariableFV.h

```diff
@@ -435,78 +432,50 @@ class MooseVariableFV : public MooseVariableField<OutputType>
    */
   ADReal getElemValue(const Elem * elem) const;
 
-  /**
-   * Compute or retrieve from cache the solution value on an internal face using linear
-   * interpolation.
-   * @param fi The face information object
-   * @return The face value on the internal face associated with \p fi
-   */
-  ADReal getInternalFaceValue(const FaceInfo & fi, const bool correct_skewness = false) const;
-
   using FunctorArg = typename Moose::ADType<OutputType>::type;
   using typename Moose::FunctorBase<FunctorArg>::ValueType;
   using typename Moose::FunctorBase<FunctorArg>::DotType;
   using typename Moose::FunctorBase<FunctorArg>::GradientType;
 
-  /**
-   * This method gets forwarded calls to \p evaluate for \p FaceArg and \p SingleSidedFaceArg
-   * spatial argument types and returns an internal face evaluation
-   */
-  template <typename FaceCallingArg>
-  ADReal getInternalFaceValue(const FaceCallingArg & face) const;
-
   void setActiveTags(const std::set<TagID> & vtags) override;
 
-  /**
-   * @return whether \p fi is an internal face for this variable
-   */
-  bool isInternalFace(const FaceInfo & fi) const override;
-
 protected:
   /**
    * @return whether \p fi is a Dirichlet boundary face for this variable
    */
-  virtual bool isDirichletBoundaryFace(const FaceInfo & fi) const;
+  virtual bool isDirichletBoundaryFace(const FaceInfo & fi, const Elem * elem) const;
```

Comment:
We need this because you may want one side of a face to be extrapolated and another be Dirichlet, e.g. when you want to allow a discontinuity in face value between - and + sides of a face. For instance the upstream value of pressure at the face is a Dirichlet condition using the extrapolated downstream pressure face value and velocity information

Reply:
Defaults can be dangerous and should be avoided most of the time. By not having a default you force the caller to think about what they are doing